### DISCUSSION TITLE
daily-backup.sh failed

### DISCUSSION THREAD

**Discussion by bgtux** [nextcloud/all-in-one#2223] [Questions] [answered]
docker exec -it -e DAILY_BACKUP=1 -e AUTOMATIC_UPDATES=1  nextcloud-aio-mastercontainer /daily-backup.sh
Daily backup script has started
Starting mastercontainer update...
(The script might get exited due to that. In order to update all the other containers correctly, you need to run this script with the same settings a second time.)
Waiting for watchtower to stop
Creating daily backup...
Waiting for backup container to stop
Starting and updating containers...
Not pulling the latest database image because the container was not correctly shut down.
Waiting for the Nextcloud container to start
Sending backup notification...
Daily backup script has finished

Backup Log:
Database export failed the last time. Most likely was the export time not high enough.
Cannot create a backup now.
Please report this to https://github.com/nextcloud/all-in-one/issues. Thanks!

### Host OS   
Debian 11

#### Nextcloud AIO version 
Nextcloud AIO v4.6.2

#### Current channel 
latest channel

**Comment by szaimen** [comment, marked as answer]
Hi, can you start the containers again and try again and check if this error happens again?
